source/obs-property.hpp

```diff
@@ -127,7 +127,15 @@ struct TextProperty : Property {
 		Password,
 		MultiLine,
 	};
+
+	enum class InfoType : uint8_t {
```

Comment:
As values will be mapped to ETextInfoType better to put values here too explicitly.